forms-flow-data-layer/src/graphql/service/submission_service.py

```diff
@@ -323,7 +323,7 @@ async def query_submissions(
                 for field, value in webapi_search.items():
                     if hasattr(application_table.c, field):
                         col = getattr(application_table.c, field)
-                        query = query.where(col == value)
+                        query = query.where(col.like(f"%{value}%"))
```

Comment:
Can we use **ilike**, which performs a case-insensitive pattern match

Reply:
we need a case-insensitive search? @arun-s-aot

Reply:
yes updated the code